### PR TITLE
Bug fix to run absolute in window git bash and cmd both.

### PR DESCRIPTION
**Pull request by cs-lee**
1. Change the "if" statement in the "download" function to determine whether
the command available.
2. Add binary path in PATH environment value for window.
3. Start script modify to determine which script used for launch in
absolute.sh.

### CODE REVIEW

**Comment by hyungheo** [comment]
looks good to me

File: bootstrap/absolute.sh [hunk 1]
```diff
@@ -10,18 +10,25 @@
 # Copy githooks to .git/hooks.
 cp $(githooks_path)/* $(absolute_path)/.git/hooks
 
-# Sync third_parties.
-sync_node
-sync_mongodb
-
 # Set path
+if is_windows_platform; then
+  if [ "$1" = "from-cmd" ]; then
```
Comment (romandev) [inline bootstrap/absolute.sh]: I don't think we need this check.
Do we really need it?

**Comment by romandev** [comment]
I left a inline comment. Please remove signed-off-by in commit description (We don't use it until now. So, you don't use git with -s option)
Also, I have two questions.
1. Does 'win-wget' is working well in both of 64bit system and 32bit system?
2. Shouldn't we have to put some license information to include the binary file in our project?

File: bootstrap/absolute.sh (hunk 1 above)
Comment (cs-lee) [inline bootstrap/absolute.sh]: That "if" statement is to export "$(third_party_path)/win-bash" path only user launch absolute in windows cmd. Without this line, bash-shell that already installed in user windows pc, run command in $(third_party_path)/win-bash.

**Pull request closed without merging**